r/tools/nixlibs.R

```diff
@@ -77,7 +77,7 @@ download_binary <- function(os = identify_os()) {
         cat("**** If installation fails, retry after installing those system requirements\n")
       }
     } else {
-      cat(sprintf("*** No C++ binaries found for %s\n", os))
+      cat(sprintf("*** No Arrow binary found for version %s on %s\n", VERSION, os))
```

Comment:
I like adding the version info into this, that's super helpful. I think we probably do want some reference to C++ or libarrow here though to be sure the person doesn't think that it's the R package binary we're talking about at this point

Reply:
Yeah, I second this, this is something I've been thinking about lately - I got tripped up with this myself when skimming a bug report.